src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/Telemetry/SearchPageTelemetryEvent.cs

```diff
@@ -18,12 +20,16 @@ public class SearchPageTelemetryEvent : TelemetryEvent
             int pageIndex,
             int resultCount,
             TimeSpan duration,
+            IEnumerable<TimeSpan> sourceTimings,
+            TimeSpan aggregationTime,
             LoadingStatus loadingStatus) : base("SearchPage")
         {
             base["ParentId"] = parentId.ToString();
             base["PageIndex"] = pageIndex;
             base["ResultCount"] = resultCount;
             base["Duration"] = duration.TotalSeconds;
+            base["SourceTimingDuration"] = string.Join(",", sourceTimings?.Select(e => e.TotalSeconds));
```

Comment:
In my Kusto adventures I have always found JSON easier to interact with than custom serialization like comma-sep. Are there VS telem conventions here?

Reply:
Not that I aware. 

I'll dig into that.